api/rest/v2/node.md

```diff
@@ -158,6 +159,7 @@ Response:
   }
 }
 ```
+
```

Comment:
please remove the empty line here. This will break the appearance.  Same for below